content/en/platform/corda/5.0/developing-applications/api/api-crypto.md

```diff
@@ -101,4 +97,25 @@ dependencies {
 You must reference `net.corda:corda-crypto-extensions`.
 {{< /note >}}
 
-<!--For information about packaging a CorDapp, see the [development tutorial]().-->
\ No newline at end of file
+## Implementing Signature Schemes
+
+Corda supports the following `SignatureSpecs` (signature schemes) for creating `java.security.Signature` objects:
```

Comment:
`SignatureSpecs` implies code and there is no such type in `corda-api` repo. Only `SignatureSpec` and `SignatureSpecService` are types visible to customers. `SignatureSpecs` is just an internal type users know nothing about.